srml/treasury/src/lib.rs

```diff
@@ -507,6 +507,35 @@ mod tests {
 		});
 	}
 
+	#[test]
+	// Note: This test demonstrates that `on_dilution` does not increase the pot with good resolution
+	// with large amounts of the network staked. https://github.com/paritytech/substrate/issues/2579
```

Comment:
The convention is to place a `TODO` and a `#number` on the same line in such cases where the code needs further work (I personally `grep` this always).